checks/nameContainsKeyword.js

```diff
@@ -0,0 +1,26 @@
+async function nameContainsKeyword(metadataJSON) {
```

Comment:
The idea behind the "keywords" check was to verify that the "keywords" property contains "aio-app-builder-template". As far as I'm concerned, we agreed not to have any restrictions about the template name.

See https://docs.npmjs.com/cli/v8/configuring-npm/package-json#keywords

Reply:
Hi Olena, I have fixed this function.